component/main.jsonnet

```
@@ -177,14 +177,43 @@ local plan = [
     ) else
       [];
 
+  local command(p) =
+    if std.objectHas(p, 'command') then (
+      if std.type(p.command) == 'string' then (
+        [ p.command ]
+      )
+      else (
+        if std.type(p.command) == 'array' then (
+          p.command
+        ) else
+          error 'Field `command` of plan "%(name)s" is not an array nor a string' % p
+      )
+    ) else
+      [];
+
+  local version = (
+    if std.objectHas(p, 'version') then
+      p.version
+    else
+      ''
+  );
+
+  local push_gateway = (
+    if std.objectHas(p, 'push_gateway') then
+      p.push_gateway
+    else
+      ''
+  );
```

```suggestion
  local push_gateway = if std.objectHas(p, 'push_gateway') then p.push_gateway;
```

We can just set `push_gateway` to `null` if it's not present in the plan configuration, since we already prune the arguments in the component library: https://github.com/projectsyn/component-system-upgrade-controller/blob/225909cd3f94f9182d674cbba0991427c41b3d01/lib/suc.libjsonnet#L31